Makefile

```diff
@@ -132,6 +132,10 @@ verify-genproto:
 verify-goimport:
 	PASSES="goimport" ./scripts/test.sh
 
+.PHONY: verify-yamllint
+verify-yamllint:
+	yamllint .github/
```

Comment:
We also have file .golangci.yaml in root directory. Can you run yamllint on whole project?

Reply:
Bump

Reply:
Addressed, thanks!